Add info. for CreatedDate and ModifiedDate properties to SDS docs (#1032)
osisoft/OCS-Docs: content/developer-guide/sequential-data-store-dev/sds-stream-views-dev.md

| SourceTypeId | String                 | Yes    | Yes		   |Identifier of the SdsType of the SdsStream |
| TargetTypeId | String                 | Yes    | Yes		   |Identifier of the SdsType to convert events to |
| Properties   | IList\<SdsStreamViewProperty\> | No    | Yes, with limitations*	  |Property-level mapping |

> | CreatedDate       | DateTime               | No    | No |  Timestamp in ISO 8601 extended format and UTC time standard at which the SdsStreamView was created. Cannot be modified by users. |

Reviewer: For all these `CreatedDate` and `ModifiedDate` entries, update the description to use "when" instead of "at which".
```suggestion
| CreatedDate       | DateTime               | No    | No |  Timestamp in ISO 8601 extended format and UTC time standard when the SdsStreamView was created. Cannot be modified by users. |
```
